.github/workflows/test.yaml

```diff
@@ -0,0 +1,21 @@
+name: Test keycloak-orgs
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        name: Checkout
+      - name: Build and Test
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+      - name: Test Organization Extension
+        run: |
+          mvn clean install
```

Comment:
add a `-B` here to keep the logs clean

also might remove the `skipTests` from the `release.yml`